hypha/apply/funds/templates/funds/includes/actions.html

```diff
@@ -14,6 +14,10 @@ <h5>Actions to take</h5>
 
 <a data-fancybox data-src="#screen-application" class="button button--bottom-space button--primary button--full-width {% if screening_form.should_show %}is-not-disabled{% else %}is-disabled{% endif %}" href="#">Screen application</a>
 
+{% if object.in_internal_review_phase or object.in_external_review_phase %}
+    {% include 'review/includes/review_button.html' with submission=object class="button--full-width button--bottom-space" extended_draft_text=True %}
```

Comment:
Consider just passing in `draft_text="Complete draft review"` and default the text to the current value.

Reply:
I've updated that, thanks